### PR TITLE
Fix errors in dashboards

### PR DESCRIPTION
**Pull request by anton-bystrov**

### CODE REVIEW

**Review by nikita-b — DISMISSED** [review]
Did you test it?

**Comment by anton-bystrov** [comment]
> Did you test it?

Yes, I tested this improvements.

File: dashboards/MySQL/PXC_Galera_Nodes_Compare.json [hunk 1]
```diff
@@ -213,7 +213,7 @@
       "links": [],
       "maxDataPoints": 100,
       "options": {
-        "colorMode": "none",
```
Comment (denisok) [inline dashboards/MySQL/PXC_Galera_Nodes_Compare.json:216]: why changes here?

File: dashboards/PostgreSQL/PostgreSQL_Instance_Summary.json [hunk 2]
```diff
@@ -227,7 +227,7 @@
       "maxDataPoints": 100,
       "options": {
         "colorMode": "none",
-        "graphMode": "graph",
```
Comment (denisok) [inline dashboards/PostgreSQL/PostgreSQL_Instance_Summary.json:230]: why we changing this?

**Review by denisok — APPROVED** [review]

**Pull request merged**